src/main/java/dev/morling/demos/logicaldecoding/Main.java

```diff
@@ -51,31 +57,26 @@ public static void main(String[] args) throws Exception {
                 .deserializer(new JsonDebeziumDeserializationSchema()) // converts SourceRecord to JSON String
                 .build();
 
-        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        // for local development, also start the Flink UI on http://localhost:8081
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(new Configuration());
```

Comment:
You won't be able to submit this jar to a cluster because we force the local environment. Would need to add a switch here.

Reply:
Check out the UI on 8081 when the JVM is running, see the checkpoints, some metrics, the job graph and other goodies.